### PR TITLE
Flags in pub fix and fix for pubchem_abstract conversion

### PR DESCRIPTION
**Pull request by ianlongden**

### CODE REVIEW

File: src/validation/yaml/publication_new.yaml [hunk 1]
```diff
@@ -157,12 +157,14 @@ P40:
 P41:
   nullable: True
   type: list
-  allowed: [RNAi, cell_cult, cell_cult::DONE, cell_line, cell_line::DONE, cis_reg, cis_reg::DONE, dataset, dataset::DONE,
+  allowed: [RNAi,cell_cult, cell_cult::DONE, cell_line, cell_line::DONE, chemical, chemical::Done, cis_reg, cis_reg::DONE,
```
Comment (gildossantos) [inline src/validation/yaml/publication_new.yaml:160]: should be `chemical::DONE` (all uppercase), not `chemical::Done`

**Review by gildossantos — CHANGES_REQUESTED** [review]
Odd - PR says 4 files changed, but it shows same file twice:
src/validation/yaml/publication.yaml

**Review by gildossantos — COMMENTED** [review]
oh - why two similar file? publication.yaml and publication_new.yaml. Should one be deleted?

**Review by gildossantos — APPROVED** [review]

**Comment by gm119** [comment]
the values you have in the two yaml files in the allowed array are not current - there are several that are no longer allowed, and some look like typos that got into the database by mistake, so they should not be treated as allowed values.
current allowed values for P41 are: (from  curation_docs/curation_manual_2.html#p41.-flag-harvard-for-curation-softcv-harvcur )

a. the following can also have ::DONE appended

wt_exp
neur_exp
gene_model
phys_int
cis_reg
genom_feat
disease
diseaseHP
diseaseF
dataset
cell_line
cell_line(commercial)
cell_line(stable)


b. allowed values for P40 are also not correct, current values allowed from cur man are:

    nocur
    nocur_abs
    merge
    split
    rename
    new_char
    new_al
    new_transg
    pheno
    gene_group
    noGOcur
    pheno_chem
    pheno_anat
    pathway

**Comment by ianlongden** [comment]
I will fix these now.

**Pull request merged**